test/upgrade/test-upgrade.bats

```diff
@@ -242,6 +245,8 @@ failed    | exited     | 17
 # if we can connect on an existing running container
 @test "network - connect" {
     skip_if_version_older 2.2.0
+    NETWORK_CONNECT_TEST=true
```

Comment:
This will not work. Each test runs in a subshell, so environment variables cannot be preserved. The usual way to work around this is by using a state file, e.g., `$PODMAN_UPGRADE_WORKDIR/ran-network-connect-test`